core-bundle/contao/dca/tl_page.php

```diff
@@ -683,7 +684,37 @@ static function ($value) {
 			),
 			'sql'                     => "int(10) unsigned NOT NULL default 0",
 			'relation'                => array('type'=>'hasOne', 'load'=>'lazy')
-		)
+		),
+		'enableCsp' => array
+		(
+			'inputType'               => 'checkbox',
+			'eval'                    => array('submitOnChange'=>true),
+			'sql'                     => array('type'=>'boolean', 'default'=>false),
+		),
+		'csp' => array
+		(
+			'inputType'               => 'text',
+			'eval'                    => array('decodeEntities'=>true),
+			'sql'                     => array('type'=>'text', 'notnull'=>false),
+		),
+		'cspLevel1Fallback' => array
+		(
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => array('type'=>'boolean', 'default'=>false),
+		),
+		'cspLegacyHeader' => array
```

Comment:
Same here - let's not introduce something that's already legacy.

Reply:
I also prefer not having this option (i.e. disabling it in general).

Reply:
Removed in 13715f0811